services/dnsmasq/dnsmasq-MacOS.sls

```diff
@@ -37,4 +33,22 @@ dnsmasq.conf:
         - user: root
         - makedirs: True
         - template: jinja
-        - source: salt://services/dnsmasq/lessonly-resolver
\ No newline at end of file
+        - source: salt://services/dnsmasq/lessonly-resolver
+
+#
+# copy the dnsmasq service plist to LaunchDaemons
+#
+/Library/LaunchDaemons/homebrew.mxcl.dnsmasq.plist:
+    file.managed:
+        - user: root
+        - makedirs: True
+        - source: /usr/local/opt/dnsmasq/homebrew.mxcl.dnsmasq.plist
+        - name: /Library/LaunchDaemons/homebrew.mxcl.dnsmasq.plist
```

Comment:
This should depend on the package being installed so that we're gauranteeed the file exists.

Reply:
Makes sense!